functions.php

```diff
@@ -52,7 +52,6 @@ function twentytwentyfour_block_styles() {
 				.is-style-arrow-icon-details {
 					padding-top: var(--wp--preset--spacing--10);
 					padding-bottom: var(--wp--preset--spacing--10);
-					border-bottom: 1px solid rgba(255, 255, 255, 0.20);
```

Comment:
I think instead of removing, we probably need to use something like currentColor

Reply:
also, I think this is not really an issue since you can choose the border color in the color picker and it will override this CSS.